### ISSUE TITLE
filepond container not scrollable

### ISSUE THREAD

**Issue by enisdenjo** [pqina/filepond#133]
### Summary

Unable to scroll when there are more items then the height of the `filepond--root`.

### How to reproduce

Set `max-height` on the `filepond--root`, upload more items than the height of the container.

### Expected behaviour

Scroll the container.

### Actual behaviour

Items go outside the container and are not scrollable.

### Environment

Filepond version: `3.0.2`

**Comment by rikschennink** [comment]
Hi, haven't tested yet, but before I before I do a quick question, have you updated both the JS and CSS of FilePond and related plugins?

**Comment by enisdenjo** [comment]
I did. I use Filepond as a module from npm.

```js
import { create } from 'filepond';
import 'filepond/dist/filepond.min.css';
```

**Comment by rikschennink** [comment]
Thanks for confirming @enisdenjo will get back to you as soon as possible.

**Comment by rikschennink** [comment]
@enisdenjo I just quickly tested this in Chrome but it seems to work correctly, did you have this problem in another browser maybe?

**Comment by rikschennink** [comment]
Okay, as I hit submit on the above message the bug jumps right at me.

**Comment by enisdenjo** [comment]
I am on Chrome too, the latest one. So you've caught the bug? 😄

**Comment by rikschennink** [comment]
We're getting there: https://twitter.com/rikschennink/status/1050274969505153025

**Comment by enisdenjo** [comment]
Haha thats right! One step closer to the fix. 😁

**Comment by rikschennink** [comment]
@enisdenjo Just published version `3.1.1` it should fix the issue. :-)

**Comment by rikschennink** [comment]
Fantastic!

Yeah, it currently doesn't redraw on resize. Feel free to open a separate issue about that.
